File: Python/0041. 简单二进制.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            n = int(input())
            res = 0
            while(n):
                res += n&1
                n >>= 1
            print(res)
        except EOFError:
            break
if __name__ == "__main__":
    func()

File: C++/0041. 简单二进制.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    int n;
    while(cin>>n){
        int res = 0;
        while(n>0){
            res += n&1;
            n >>= 1;
        }
        cout<<res<<endl;
    }
    return 0;
}
